scripts/http-vuln-cve2014-3704.nse

```diff
@@ -215,10 +215,12 @@ local function do_sql_query(host, port, uri, user)
       ['Content-Type'] = "application/x-www-form-urlencoded"
     }
   }
-  local res = http.post(host, port, uri .. "/user/login", opt, nil, r)
-  --TODO: Check return status
+  local res = http.post(host, port, uri .. "?q=/user/login", opt, nil, r)
+
+  if string.match(res.body, ">Have you forgotten your password?") then
```

Comment:
This condition unfortunately does not match every installation, because the string is translated. The source (`modules/user/user.module`) is here:

```php
form_set_error('name', t('Sorry, unrecognized username or password. <a href="@password">Have you forgotten your password?</a>', array('@password' => url('user/password'))));
```

The `t()` function call translates it. In Spanish, it translates to: `Lo sentimos. No reconocemos el nombre de usuario o la contraseña. <a href="@password">¿Olvidó su contraseña?</a>`

Is there some other thing we could match on? Maybe the URL to the password reset page or something? You could email me the source of the page that results and I can try to help come up with something.

In any case, we should put a debug message here if this match does not succeed.